### ISSUE TITLE
how many epochs I need at least? 

### ISSUE THREAD

**Issue by xxoospring** [JusperLee/Dual-Path-RNN-Pytorch#9]
how many epochs when do your own training? Now I've run two epoch and loss is -16.8.

**Comment by JusperLee** [comment]
https://github.com/JusperLee/Dual-Path-RNN-Pytorch/blob/master/log/Conv_Tasnet/loss.png
